src/tests/responses/bullyisms.test.ts

```diff
@@ -24,4 +24,24 @@ describe('bullyisms', () => {
       bullyTemplate.isTriggered('this sentence shouldnt trigger')
     ).toBeFalsy()
   })
+
+  test('millenial rage', () => {
+    expect(
+      millenialRage.isTriggered(
+        'back in my #day'
+      )
+    ).toBeTruthy()
+
+    expect(
+      millenialRage.isTriggered(
+        '<cfoutput>#cfcatch#</cfoutput>'
+      )
+    ).toBeTruthy()
```

Comment:
Shouldn't this be a situation we'd like to weed out or are we intentionally having OldMan think this is a valid hashtag?

Reply:
1) The old man doesn't know the difference between ColdFusion trash and social media hashtags.
2) I'm bad at regex.

Reply:
I'm fine with either justification. LG2M